components/calendar/views/days.js

```diff
@@ -0,0 +1,305 @@
+    const { startWeekDay } = locale;
+
+    /** Calculates the amount of days that  */
+    const currentDate = new Date(renderDate.getFullYear(), renderDate.getMonth(), 1, 0, 0, 0);
+    const firstWeekDayOfMonth = currentDate.getDay();
+    const numDaysOfPreviousMonth = firstWeekDayOfMonth < startWeekDay ? 6 : (firstWeekDayOfMonth - startWeekDay);
+
+    currentDate.setDate(currentDate.getDate() - numDaysOfPreviousMonth);
+
+    const options = [];
+    let counter = 0;
+
+    const limits = selectedDates.map(item => (item ? item.getTime() : null));
+    const selectedDatesAreEqual = (limits.length === 2) && (limits[0] === limits[1]);
+
+    while (counter < 42) {
+      const extraClasses = [];
+      const selectedDate = selectedDates && selectedDates.find((item) => {
+        return item && (item.toDateString() === currentDate.toDateString());
+      });
+
+      if (currentDate.getMonth() < renderDate.getMonth()) {
+        extraClasses.push('previous-month');
+      } else if (currentDate.getMonth() > renderDate.getMonth()) {
+        extraClasses.push('next-month');
+      }
+
+      if (selectedDate) {
+        if ((selectionType === CALENDAR_SELECTION_TYPE_RANGE) && !selectedDatesAreEqual) {
+          extraClasses.push((selectedDates.indexOf(selectedDate) === 0) ? 'selected-start' : 'selected-end');
+        } else {
+          extraClasses.push('selected');
+        }
+      } else if (limits[1] && (limits[0] <= currentDate.getTime()) && (limits[1] >= currentDate.getTime())) {
+        extraClasses.push('selected-between');
+      }
+
+      const className = getClassNamesWithMods('ui-calendar-days-option', extraClasses);
+
+      const onClickHandler = onSelectDay ? onSelectDay.bind(null, new Date(currentDate.toDateString())) : undefined;
+      options.push(
+        <button
+          className={className}
+          disabled={!this.isOptionEnabled(currentDate)}
+          key={`option_${counter}`}
+          onClick={onClickHandler}
+          type="button"
+        >{currentDate.getDate()}</button>
+      );
+      currentDate.setDate(currentDate.getDate() + 1);
+      counter += 1;
+    }
+
+    return <div className="ui-calendar-days__options">{options}</div>;
+  }
+
+  /**
+   * Renders the navigation of the days' calendar.
+   *
+   * @method renderNav
+   * @return {HTMLElement}
+   */
+  renderNav() {
+    const { renderDate, locale, minDate, maxDate, onNavPreviousMonth, onNavNextMonth } = this.props;
+
+    const isPreviousMonthDisabled = minDate && (minDate.getMonth() >= renderDate.getMonth());
+    const isNextMonthDisabled = maxDate && (maxDate.getMonth() <= renderDate.getMonth());
+
+    return (
+      <nav className="ui-calendar-days__nav">
+        <button
+          className="ui-calendar-days__previous-month"
+          disabled={isPreviousMonthDisabled}
+          onClick={onNavPreviousMonth}
+          type="button"
+        >&lt;</button>
```

Comment:
Yes :) That's my next challenge 😄 ... Probably will expose via props what you can pass as nav buttons (next and previous).